jupyterdrive/__init__.py

```diff
@@ -57,7 +60,17 @@ def main(argv=None):
     prog = '{} -m jupyterdrive'.format(os.path.basename(sys.executable))
     parser = argparse.ArgumentParser(prog=prog,
                     description='Install Google Drive integration for Jupyter.')
-    parser.add_argument('profile', nargs='?', default='default')
+    parser.add_argument('profile', nargs='?', default=None, metavar=('<profile_name>'))
+    parser.add_argument("-m", "--mixed", help="Installed the mixed content manager",
+                    action="store_true")
+    parser.add_argument("-S", "--no-symlink", help="do not symlink at install time",
+                    action="store_false")
+    parser.add_argument("-u", "--user", help="force install in user land",
+                    action="store_true")
     args = parser.parse_args(argv)
-    install(profile=args.profile)
-
+    if not args.profile:
+        parser.print_help()
+        sys.exit(0)
+    install(profile=args.profile,
+              mixed=args.mixed,
+            symlink=args.no_symlink)
```

Comment:
Should it ? cause above it's `store_false` ? I'll re-check. I can try set metavar to be more readable.